src/fields_parse.rs

```diff
@@ -12,15 +12,18 @@ pub(crate) enum ParserType {
 
 impl ParserType {
     pub fn parse(value: AttrValue) -> Result<Self> {
-        value.expect(r#""default", "split" or a path"#, |v| match v {
-            AttrValue::Path(p) => Ok(ParserType::Custom(p)),
-            AttrValue::Lit(syn::Lit::Str(ref l)) => match &*l.value() {
-                "default" => Ok(ParserType::Default),
-                "split" => Ok(ParserType::Split { separator: None }),
+        value.expect(
+            r#""default", "split", or a path for custom parsers"#,
```

Comment:
```suggestion
            r#""default", "split", or a path to a custom parser function"#,
```

Maybe this wording is clearer?